indico/modules/events/surveys/fields/base.py

```diff
@@ -50,6 +50,8 @@ class SurveyField(object):
     plugin = None
     #: displayed name of the field type
     friendly_name = None
+    #: wtform field class for this field
+    wtf_field_class = None
```

Comment:
why do we define this here if the base class doesn't use it at all? i'd rather add a degault `wtf_field` implementation that uses it:

    return self._make_wtforms_field(self.wtf_field_class, self.validators)

That way we have a sane default and people can still override it if necessary